azure/Guidelines.md

```diff
@@ -704,30 +836,36 @@ Azure services need to change over time. However, when changing a service, there
 
 *NOTE: the [Azure Breaking Change Policy](http://aka.ms/AzBreakingChangesPolicy/) has tables (section 5) describing what kinds of changes are considered breaking. Breaking changes are allowable (due to security/compliance/etc.) if approved by the [Azure Breaking Change Reviewers](mailto:azbreakchangereview@microsoft.com) but only following ample communication to customers and a lengthy deprecation period.*
 
+<a name="versioning-review-required"></a>
 :white_check_mark: **DO** review any API changes with the Azure API Stewardship Board
 
 Clients specify the version of the API to be used in every request to the service, even requests to an `Operation-Location` or `nextLink` URL returned by the service.
 
+<a name="versioning-api-version-query-param"></a>
 :white_check_mark: **DO** use a required query parameter named `api-version` on every operation for the client to specify the API version.
 
+<a name="versioning-date-based-versioning"></a>
 :white_check_mark: **DO** use `YYYY-MM-DD` date values, with a `-preview` suffix for preview versions, as the valid values for `api-version`.
 
 ```text
 PUT https://service.azure.com/users/Jeff?api-version=2021-06-04
 ```
 
+<a name="versioning-use-later-date"></a>
 :white_check_mark: **DO** use a later date for each new preview version
 
 When releasing a new preview, the service team may completely retire any previous preview versions after giving customers at least 90 days to upgrade their code
 
-:white_check_mark: **DO** use a later date for successive preview versions.
-
```

Comment:
I removed this rule as it looked like a duplicate of "versioning-use-later-date" on line 855 above.